include/odp/api/spec/timer.h

```diff
@@ -140,6 +140,8 @@ typedef struct {
 	/** Clock source for timers */
 	odp_timer_clk_src_t clk_src;
 
+	/** Frequency of the constant clock source. */
+	uint64_t const_hz;
```

Comment:
We believe that this parameter is not really helpful as it needs to be known while calling `odp_timer_capability` and `odp_timer_res_capability`  functions, It is the responsibility of the implementation to figure out the EXT_CLK frequency.
Will drop this and send a v7.